tests/helpers/test_entity_platform.py

```diff
@@ -46,6 +46,7 @@ async def test_polling_only_updates_entities_it_should_poll(
 ) -> None:
     """Test the polling of only updated entities."""
     component = EntityComponent(_LOGGER, DOMAIN, hass, timedelta(seconds=20))
+    await component.async_setup({})
```

Comment:
This is the first type of change in the file.

Tests that create an instance of `EntityComponent` need to call `async_setup` so that it starts listening for STOP event from home assistant, which will trigger an automatic cleanup.